common/locales/en/partyLoadingMessages.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Adding new files to Transifex is kind of a pain. I think these can just live in the groups json file for now.